docs/FirstUsage.md

```diff
@@ -0,0 +1,148 @@
+# First Usage
+
+The document describes how to setup first simple project. We will setup build and deploy process, show basic project structure. Make sure you have installed [psake](https://github.com/psake/psake) and [PSGallery](GettingStarted.md) into your project.
+
+## Create First Task
+
+You may find `default.ps` file and `scripts` directories in the root of your project. `default.ps1` is used for Psake task runner and it is an entry point. It contains project properties and includes all main configuration. `scripts` directory contains PSGallery modules and predefined Psake tasks. Example:
+
+```
+./
+scripts/          // Psake modules and build files.
+src/              // Application.
+default.ps1       // Entry point for Psake.
+```
+
+Let's overview Psake entry point with comments:
+
+```powershell
+# What .NET framework should Psake use.
+Framework 4.6
+$InformationPreference = 'Continue'
+$env:PSModulePath += ";$PSScriptRoot\scripts\modules"
+
+# Include modules, we have them locally.
+. .\scripts\Saritasa.GitTasks.ps1
+. .\scripts\Saritasa.PsakeExtensions.ps1
+. .\scripts\Saritasa.PsakeTasks.ps1
+
+# Our custom tasks. When we have a lot of tasks it is better to separate them by files.
+. .\scripts\BuildTasks.ps1
+. .\scripts\PublishTasks.ps1
+
+# Global properties.
+Properties `
+{
+
+
+    $Configuration = 'Debug'
+    $Environment = 'Development'
+
+
+}
+```
+
+We can start by creating simple task in `scripts\BuildTasks.ps1` file. Add following content to the end of file:
+
+```powershell
+Task test `
+    -description 'Test task.' `
+    -requiredVariables @('Param') `
+{
+    Write-Information("You are using configuration $Configuration with param value $Param.");
```

Comment:
Write-Information "You are using configuration $Configuration with param value $Param."